src/views/ApartmentApp/components/StudentApplication/components/OffCampusSection/components/ProgramListItem/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Divider,
+  FormControl,
+  Grid,
+  Input,
+  InputLabel,
+  ListItem,
+  ListItemText,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+const ProgramListItem = ({
+  disabled,
+  index,
+  applicantProfile,
+  offCampusProgram,
+  departments,
+  onOffCampusInputChange,
+}) => {
+  const [applicantDepartmentValue, setDepartmentValue] = useState(''); // Department drop-down menu value
+
+  useEffect(() => {
+    setDepartmentValue(offCampusProgram ?? '');
+  }, [offCampusProgram]);
```

Comment:
This use of the state variable is not necessary, as you can simply pass the prop to the `Select` component directly. Evan gave a great explanation of this is another PR, but I do not know which one and I was unable to find it
```suggestion
```

Reply:
You can also see the newest version of the Hall components, which have been greatly improved since the last time you worked on this feature